cachito/workers/pkg_managers/pip.py

```diff
@@ -8,6 +8,7 @@
 import secrets
 import shutil
 import tarfile
+from typing import Optional
```

Comment:
BTW are we going to add isort check?

Reply:
sounds good, but how do you want to organize both `black` and `isort`?

Reply:
I don't have such plan, I never used isort. I just remember that fiting pylint import order requirements was a pain, sometime impossible :D

Reply:
But feel free to add it